<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgPartSelect</class>
 <widget class="QDialog" name="dlgPartSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>661</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblPart">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Part:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtPart">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbPart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>671</width>
     <height>581</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>540</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>600</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNew">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>360</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtName">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>130</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtSpec">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>220</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lbSpec">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>180</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Specitional:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtType">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>310</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblType">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>270</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>480</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtSearch">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>420</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
